source/slang/slang-compiler.cpp

```diff
@@ -2471,6 +2471,18 @@ namespace Slang
         return nullptr;
     }
 
+    SLANG_NO_THROW SlangInt32 SLANG_MCALL Module::getDependencyFileCount()
+    {
+        return (SlangInt32)getFileDependencies().getCount();
+    }
+
+    SLANG_NO_THROW char const* SLANG_MCALL Module::getDependencyFilePath(
+        SlangInt32 index)
+    {
+        SourceFile* sourceFile = getFileDependencies()[index];
+        return sourceFile->getPathInfo().hasFoundPath() ? sourceFile->getPathInfo().foundPath.getBuffer() : "unknown";
```

Comment:
I think there could be an actual file called "unknown" for a certain situation.
Probably it is better with an empty string, "".

Reply:
Yeah, it may be better to just return nullptr.

Reply:
Done.